### DISCUSSION TITLE
From 'text'

### DISCUSSION THREAD

**Discussion by WahyuAgungBudiyanto** [simple-login/app#1605] [Q&A]
![image](https://user-images.githubusercontent.com/73398631/221355297-89370c16-f557-4c4a-8e31-4252db77d47c.png)
Is there a way to hide this simplelogin.co

**Comment by WahyuAgungBudiyanto** [comment]
I did all the DNS verification but still

**Reply by nguyenkims** [reply]
All emails forwarded by SimpleLogin are sent by simplelogin.co domain which is used to create reverse alias.

**Reply by WahyuAgungBudiyanto** [reply]
so in that case i cant hide it? to make it look clean or like pretend not using simplelogin?

**Reply by WahyuAgungBudiyanto** [reply]
I just want to forward to certain email but i dont want them to know im using simple login

**Reply by WahyuAgungBudiyanto** [reply]
sir?

**Reply by rehanone** [reply]
Simplelogin is a service that hides your "REAL" email from the recipient by exposing an alias to the recipient. If you don't want your recipient to know about Simplelogin as well then the best way would be to never send them an email. There is no such thing as anonymous email which I believe you are try to get at.
